docs/modules/ROOT/pages/hazelcast-platform-operator-jet.adoc

```diff
@@ -0,0 +1,339 @@
+
++
+Replace the <BUCKET-URI> with the appropriate sample bucket URI for your cloud provider:
++
+- s3://hazelcast-jet-pipeline
+- gs://hazelcast-jet-pipeline
+- azblob://hazelcast-jet-pipeline
+
++
+[source, shell]
+----
+cat <<EOF | kubectl apply -f -
+apiVersion: hazelcast.com/v1alpha1
+kind: JetJob
+metadata:
+  name: my-jet-job
+spec:
+  hazelcastResourceName: my-hazelcast
+  state: Running
+  jarName: jet-pipeline-run-1.0.0.jar
+  bucketConfig:
+    bucketURI: "<BUCKET-URI>"
+    secretName: <SECRET-NAME>
+EOF
+----
+
+. Check status of the Jet job.
+
++
+[source,bash]
+----
+kubectl get jetjob my-jet-job -w
+----
++
+The output will look something like this. Wait until the `Running` status is displayed.
++
+[source,bash]
+----
+NAME         STATUS    ID                   SUBMISSIONTIME         COMPLETIONTIME
+my-jet-job   Starting  0
+my-jet-job   Running   732187341048774657   2023-07-14T10:51:06Z
+----
+
+. To see the output of the Jet pipeline in the logs, run the following command.
+
++
+[source,bash]
+----
+kubectl logs -l app.kubernetes.io/name=hazelcast -c hazelcast | grep sequence
+----
++
+The logs of Jet Pipeline is displayed in the output.
++
+[source,bash]
+----
+{"time":"2023-07-14T10:51:06,679", "logger": "com.hazelcast.jet.impl.connector.WriteLoggerP", "level": "INFO", "msg": "[172.17.0.6]:5702 [dev] [5.3.1] [my-jet-job/loggerSink#0] SimpleEvent(timestamp=10:51:06.000, sequence=0) "}
+{"time":"2023-07-14T10:51:07,675", "logger": "com.hazelcast.jet.impl.connector.WriteLoggerP", "level": "INFO", "msg": "[172.17.0.6]:5702 [dev] [5.3.1] [my-jet-job/loggerSink#0] SimpleEvent(timestamp=10:51:07.000, sequence=1) "}
+{"time":"2023-07-14T10:51:08,681", "logger": "com.hazelcast.jet.impl.connector.WriteLoggerP", "level": "INFO", "msg": "[172.17.0.6]:5702 [dev] [5.3.1] [my-jet-job/loggerSink#0] SimpleEvent(timestamp=10:51:08.000, sequence=2) "}
+{"time":"2023-07-14T10:51:09,679", "logger": "com.hazelcast.jet.impl.connector.WriteLoggerP", "level": "INFO", "msg": "[172.17.0.6]:5702 [dev] [5.3.1] [my-jet-job/loggerSink#0] SimpleEvent(timestamp=10:51:09.000, sequence=3) "}
+{"time":"2023-07-14T10:51:10,682", "logger": "com.hazelcast.jet.impl.connector.WriteLoggerP", "level": "INFO", "msg": "[172.17.0.6]:5702 [dev] [5.3.1] [my-jet-job/loggerSink#0] SimpleEvent(timestamp=10:51:10.000, sequence=4) "}
+{"time":"2023-07-14T10:51:11,678", "logger": "com.hazelcast.jet.impl.connector.WriteLoggerP", "level": "INFO", "msg": "[172.17.0.6]:5702 [dev] [5.3.1] [my-jet-job/loggerSink#0] SimpleEvent(timestamp=10:51:11.000, sequence=5) "}
+{"time":"2023-07-14T10:51:12,677", "logger": "com.hazelcast.jet.impl.connector.WriteLoggerP", "level": "INFO", "msg": "[172.17.0.6]:5702 [dev] [5.3.1] [my-jet-job/loggerSink#0] SimpleEvent(timestamp=10:51:12.000, sequence=6) "}
+....
+....
+----
+
+. Now, cancel the Jet job by setting the `spec.state` to `Canceled`. Use one of the following options.
+
++
+a) Run the following command to cancel the JetJob.
```

Comment:
```suggestion
a) Run the following command to cancel the JetJob.
```
Don't think we can have two options that say exactly the same thing. Needs to say something like 'Run the following command to cancel the JetJob, using ????.

Reply:
Yes you are right? Let me update them.